contracts/RootChain.sol

```diff
@@ -111,6 +111,19 @@ contract RootChain {
         uint256 _spendingTxSignatures,
         uint256 _spendingTxConfirmationSignatures
     ) public {
+		uint256 eUtxoPos = _encodedSpendingTx.getUtxoPos(_exitingUtxoPosition);
+
+        uint256 _txindex  = (_spendingTxPosition % 1000000000) / 10000;
```

Comment:
We can make use of `PlasmaCore` here instead:
`uint256 _txindex = getTxIndex(_spendingTxPosition)`

Likewise for `root`.